go/tasks/pluginmachinery/service/properties.go

```diff
@@ -0,0 +1,39 @@
+package service
+
+import "time"
+
+// The plugin manager automatically queries the remote API
+type RateLimiterProperties struct {
```

Comment:
We should probably have separate rate limiters for `get` and `create`

Reply:
good idea